src/tools/tests/schema2kotlin-test.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * @license
+ * Copyright (c) 2020 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+
+import {assert} from '../../platform/chai-node.js';
+import {KotlinGenerator} from '../schema2kotlin.js';
+import {SchemaNode} from '../schema2graph.js';
+import {Schema} from '../../runtime/schema.js';
+
+
+describe('schema2wasm', () => {
```

Comment:
Not done?

Reply:
I had an in-person discussion with Walter about this. The area of code is schema2wasm. I intend for these tests to be run when the sigh tool is also run. 

A further note: schema2wasm is not a good name for the tool, since it doesn't do stuff just for wasm. 

The reason why I have two describes is so that I can control testing the inner describe or the outer.